src/Symfony/Component/Security/Core/Resources/translations/security.en.xlf

```diff
@@ -70,6 +70,14 @@
                 <source>Invalid or expired login link.</source>
                 <target>Invalid or expired login link.</target>
             </trans-unit>
+            <trans-unit id="19">
```

Comment:
If the only difference between the two message is the singular/plural "minutes" ... shouldn't we use the pluralization feature of translation messages?

Reply:
I think the error message comes from here: https://github.com/symfony/symfony/blob/v5.3.0-BETA1/src/Symfony/Component/Security/Core/Exception/TooManyLoginAttemptsAuthenticationException.php#L44

So maybe it would require updating the Exception first.

Reply:
Yes, we should fix the message key. See my comment on the issue.